echopype/convert/set_groups_azfp.py

```diff
@@ -789,6 +818,10 @@ def set_vendor(self) -> xr.Dataset:
                     list(range(len(unpacked_data["ancillary"][0]))),
                 ),
                 "ad_len": (["ad_len"], list(range(len(unpacked_data["ad"][0])))),
+                "phase_number": (
+                    ["phase_number"],
+                    sorted([int(num) for num in parameters["phase_number"]], reverse=False),
```

Comment:
`reverse=False` is unnecessary, since that's the default.
```suggestion
                    sorted([int(num) for num in parameters["phase_number"]]),
```